### ISSUE TITLE
Build failure from source

### ISSUE THREAD

**Issue by D3vil0p3r** [mborgerson/pyside6_qtads#44]
Hey @mborgerson I'm trying to build the library from source by using:
```
git submodule update --init --recursive
python -m build --wheel --outdir="$startdir/dist"
```
During the build I get this error:
```
CMake Error at CMakeLists.txt:238 (target_compile_options):
  Cannot specify compile options for target "qt6advanceddocking" which is not
  built by this project.
```
Is there an upstream bug on identifying the target?

**Comment by mborgerson** [comment]
Not sure about this build issue. Did the submodule actually clone correctly?

**Comment by D3vil0p3r** [comment]
Yes, because if I don't get submodule, I get a different error.

Did you try to build it from source on Arch Linux?

**Comment by mborgerson** [comment]
> Did you try to build it from source on Arch Linux?

Nope. The wheel is built on manylinux (alma). Maybe an issue with cmake.

See [here](https://github.com/mborgerson/pyside6_qtads/blob/main/.github/workflows/main.yml#L184) for how it's built on CI for debugging.
